qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -166,3 +171,78 @@ def tearDown(self):
                                        omit_sudo=False, check_status=False)
 
         super(XFSTestsDev, self).tearDown()
+
+
+    #
+    # Following are helper methods that launch individual and groups of tests
+    # from xfstests-dev repo that is ready.
+    #
+
+    # generic helper methods
+
+    def run_tests(self, cmdargs):
+        cmd = 'sudo ./check ' + cmdargs
+        p = self.mount_a.run_shell(args=cmd, cwd=self.repo_path,
+            stdout=StringIO(), stderr=StringIO(), check_status=False,
+            omit_sudo=False, timeout=180)
+
+        if p.returncode != 0:
+            log.info('Command failed')
+        log.info(f'Command return value: {p.returncode}')
+        stdout, stderr = p.stdout.getvalue(), p.stderr.getvalue()
+        log.info(f'Command stdout -\n{stdout}')
+        log.info(f'Command stderr -\n{stderr}')
+
+        self.assertEqual(p.returncode, 0)
+        line = 'Passed all 0 tests' # unexpected failure line
+        self.assertNotIn(line in stdout)
+
+        return p.returncode
+
+    def run_testfile(self, testdir, testfile):
+        return self.run_tests(f'{testdir}/{testfile}')
```

Comment:
Why not fix the `test_acls.py` by using this new helper in this PR?

Reply:
I intend to do it but I wanted to figure out first what more tests we want to run.